src/main/java/com/amazon/opendistro/elasticsearch/performanceanalyzer/decisionmaker/actions/ModifyQueueCapacityAction.java

```diff
@@ -119,6 +121,17 @@ public ResourceEnum getThreadPool() {
     return threadPool;
   }
 
+  //persist values into SQL tuple
+  private void persistValues() {
+    setRowValue(ActionField.RESOURCE_TYPE_FIELD.getField(), threadPool.getNumber());
+    setRowValue(ActionField.NODE_ID_FIELD.getField(), esNode.getNodeId().toString());
+    setRowValue(ActionField.NODE_IP_FIELD.getField(), esNode.getHostAddress().toString());
+    setRowValue(ActionField.ACTIONABLE_FIELD.getField(), isActionable());
+    setRowValue(ActionField.COOL_OFF_PERIOD_FIELD.getField(), coolOffPeriodInMillis);
+    setRowValue(ActionField.CURRENT_VALUE_FIELD.getField(), currentCapacity);
+    setRowValue(ActionField.DESIRED_VALUE_FIELD.getField(), desiredCapacity);
```

Comment:
These, along with the `RESOURCE_TYPE_FIELD` are very specific to resource tuning actions. The schema wont scale to other types of actions. Instead, how about we just have a `summary` column in the table, which takes a json for each action's summary. The action can define its summary in whatever terms are suitable to it. 

Additionally, each ActionBuilder can have accept a `summary(Object obj)` field that can parse the summary to recreate its corresponding action (to allow consumers of this table to recreate actions from generic summary jsons).